mirascope/core/openai/__init__.py

```diff
@@ -14,7 +16,9 @@
 from .stream import OpenAIStream
 from .tool import OpenAITool, OpenAIToolConfig
 
-OpenAIMessageParam: TypeAlias = ChatCompletionMessageParam | BaseMessageParam
+OpenAIMessageParam: TypeAlias = (
+    ChatCompletionMessageParam | BaseMessageParam | AudioSegment | Wave_read
```

Comment:
this seems wrong? `OpenAIMessageParam` should only be `ChatCompletionMessageParam | BaseMessageParam` since audio/image etc. inputs/outputs should be converted to message parameters by the time this type is every used

Reply:
you are right. I fixed it.